src/mr-importer/optimizer.cpp

```
@@ -12,219 +12,277 @@
-        "Mesh has no attributes, but they are considered by `optimize` function."
-        " Consider adding attribute-less path in optimize"
-      );
-      mesh.attributes.resize(mesh.positions.size());
-    }
+  Mesh result;
+  result.transforms = std::move(mesh.transforms);
+  result.name = std::move(mesh.name);
+  result.aabb = mesh.aabb;
+  result.material = mesh.material;
+
+  std::array streams = {
```

Clang (14.0.6)

[warning] variable 'streams' is not initialized

(cppcoreguidelines-init-variables)